.github/workflows/cache.yml

```diff
@@ -0,0 +1,26 @@
+name: Cache
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/cache@v1.1.2
+      id: cache
+      if: startsWith(runner.os, 'Linux')
```

Comment:
All of the jobs run on Linux right now and I don't foresee us adding MacOS Github Actions any time soon, so we can probably omit the `startsWith` check.